src/app/qgisapp.cpp

```diff
@@ -14420,8 +14420,8 @@ void QgisApp::showMapTip()
     QgsMapLayer *mypLayer = mMapCanvas->currentLayer();
     if ( mypLayer )
     {
-      // only process vector layers
-      if ( mypLayer->type() == QgsMapLayerType::VectorLayer )
+      // only process vector and raster layers
+      if ( mypLayer->type() == QgsMapLayerType::VectorLayer || mypLayer->type() == QgsMapLayerType::RasterLayer )
```

Comment:
couldn't we just always get the map tip template value, and if it's non-empty then show?

Reply:
Yes, the check can be avoided.